### DISCUSSION TITLE
foot window do not swallow

### DISCUSSION THREAD

**Discussion by moilong** [hyprwm/Hyprland#1513] [Q&A] [answered]
` misc {                                                                                                                           
  enable_swallow = true
  swallow_regex = ^(foot)$
  }`
with the above conf if i open a video file in mpv foot window stays visible along with mpv. What am I doing wrong.

**Comment by vaxerski** [comment, marked as answer]
umm I assume the missing newlines are an issue with you pasting, it should look like
```
misc {
    enable_swallow = true
    swallow_regex = ^(foot)$
}
```

If that doesn't work then it's a problem with the fact that foot spawns all windows with the same pid, so it's probably interfering with the detection. See https://github.com/hyprwm/Hyprland/issues/772

**Reply by moilong** [reply]
config is exactly like you pasted. Dont know why it looks as single line when i paste. I will try kitty or alacritty.

**Reply by moilong** [reply]
Starting foot as client `exec, footclient` solves the issue. Thanks for pointing to https://github.com/hyprwm/Hyprland/issues/772.
